### DISCUSSION TITLE
[copilot.vim] Unable to type capital `Ü` umlaut in input mode

### DISCUSSION THREAD

**Discussion by cbix** [community/community#53674] [Copilot Conversations] [answered]
### Select Topic Area

Bug

### Body

I started using the copilot.vim plugin with vim last week and just noticed that even in input mode, I can't type the capital letter `Ü` anymore. Lower caps and other umlauts Ä/Ö are working just fine and disabling the copilot plugin solves the issue.

Since we need to write a lot of UI code for german-speaking customers, this is a rather big issue and I'll stop using Copilot until this issue is resolved or I find a workaround.

**Comment by tpope** [comment, marked as answer]
This is fixed in Copilot.vim 1.8.4.
